dashboards/mobile/mobile.json

```diff
@@ -1,308 +1,238 @@
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 8,
+            "height": 5
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Welcome to the New Relic Mobile dashboard! 📱📊\n## Before you start, make sure to follow this little instruction for maximum visibility. After that, you may delete this text box.\n\nThe views in this dashboard are `entity-guid` dependant, so it's important you add it to all your charts' NRQL queries. But fear no more, this is easy as 🍰! Complete these checklists and you'll be ready to go.\n\n## Finding your Entity guid:\n- [ ] Navigate to the [Mobile](https://one.eu.newrelic.com/mobile)  section.\n- [ ] Find your app and click the ⋮ button on the right.\n- [ ] Click on *See metadata & tags* and copy your `entityGuid`.\n\n## Editing your NRQL queries:\n\n- [ ] Go to your charts and click the ⋮ button on the top right corner.\n- [ ] Click on *Edit* and add replace `addYourOwnGuid` with the guid you found. Make sure it remains \"in quotations\".\n\n > NOTE: While some of the charts in this dashboard might work out of the box without this addition, please bear in mind if you don't specify an Entity guid the results displayed will be an aggreagation of *all* your mobile apps in your account."
+          }
+        },
+        {
+          "title": "App Version",
+          "layout": {
+            "column": 9,
+            "row": 1,
+            "width": 4,
+            "height": 3
           },
-          {
-            "title": "Installations by country",
-            "layout": {
-              "column": 5,
-              "row": 1,
-              "width": 4,
-              "height": 3
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.pie"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": true
             },
-            "linkedEntityGuids": null,
-            "visualization": {
-              "id": "viz.pie"
+            "legend": {
+              "enabled": true
             },
-            "rawConfiguration": {
-              "facet": {
-                "showOtherSeries": true
-              },
-              "legend": {
-                "enabled": true
-              },
-              "nrqlQueries": [
-                {
-                  "accountId": 0,
-                  "query": "SELECT uniqueCount(uuid) FROM MobileRequest FACET `countryCode` SINCE 1 month ago EXTRAPOLATE"
-                }
-              ],
-              "platformOptions": {
-                "ignoreTimeRange": true
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT uniqueCount(appVersion) FROM MobileRequest WHERE (entityGuid = 'addYourOwnGuid') FACET appVersion SINCE 1 month ago EXTRAPOLATE"
```

Comment:
The user would need to install this dashboard then go in and update this `entityGuid` to get the dashboard working. Overall, we prefer these dashboards to work out of the box, is there a way to do this without needing the entity guid? I've seen a lot of other authors use `FACET appName` to handle this type of case.

Reply:
Ignore me, the dashboard has an explainer at the top telling users how to update the NRQL.